### DISCUSSION TITLE
Can somebody show me how to put a login page that works online.

### DISCUSSION THREAD

**Discussion by lawchin** [flet-dev/flet#1473] [General]
Hello, please be informed that I am NOT looking for tutorial on how to create a login page. I just want to learn on how to put the created login page that works online.

Does anybody here have it done already? can you please share link and login/password so I can try login.

I have one question about static page, Does login page works on static web app? if it does, how did you make it work?.

p/s:  been looking for answer for almost 5 months now

**Comment by lawchin** [comment]
anyone?

**Comment by Godmemer420** [comment]
Hey, A free resource to do this is oracle cloud. They offer a free tier service, you will need to link a card but you will not be charged, just look at some youtube tutorial. Then create a ubuntu VM and SSH into it. copy your flet app into the server and make sure that you got this:
`ft.app(target=TARGET, view=ft.WEB_BROWSER, port=PORT)`
TARGET is well you target and port is any port choosen by you.
Now you can go and install nginx and change adjust the default files to your needs.
View here under **NGINX Proxy Setup** https://flet.dev/docs/guides/python/deploying-web-app/hosting-providers/self-hosting
On youtube you can look for **NGINX reverse proxy setup** and follow any tutorials to deploy your app.
Please make sure to open port 80 as your NGINX will be http first, if you install a certificate then open port 433 too. Also open the **PORT** that you defined in your flet app.
You can open the ports in the cloud oracle web interface or just look at a tutorial on youtube for that too.

Now you got yourself a free VPS and deployed your flet app on it.

**Reply by lawchin** [reply]
do I still hit "flet publish main.py" and get the 'dist' folder? I dont think I understand this.. but thanks anyway.

**Reply by Godmemer420** [reply]
in this case not, you would put the python file online but you can probably do the same with flet publish.
